### ISSUE TITLE
Why is the button labeled "Space Port"...

### ISSUE THREAD

**Issue by Amazinite** [endless-sky/endless-sky#2669]
when in all 247 instances that I could fine it used in missions, it's written as "Spaceport"? There are only 6 instances in the game of it being called a "Space Port," 4 of which are in the intro missions, 1 is in a help dialog, and the last place is the button itself being called the Space Port. Everywhere else the words are combined. For good old consistency's sake, can we make it one word in all instances?

**Comment by Startome** [comment]
To be honest, any planet which has a spaceport built, that's probably where the player lands.  I've just always thought it was silly to go to the spaceport when your player just landed at tfw spaceport. Maybe that button should get a name change instead?  
Examples: 
Lobby
Lounge
Bar
Space Bar
Visitor Center
43
Observation Deck
Hospitality Zone

**Comment by Startome** [comment]
Tfw your phone auto corrects "the" to "tfw"

**Comment by Amazinite** [comment]
Space Hub? There's more than just the bar in a spaceport and probably only a few worlds have observation decks.

**Comment by Bladewood** [comment]
Hub?

**Comment by Startome** [comment]
Google.com ---> "hub"

**Comment by Amazinite** [comment]
hub (noun): the effective center of an activity, region, or network.
e.g. The spaceport is the hub of each world.

**Comment by Bladewood** [comment]
No, I know what "hub" means. I'm not an idiot.
I was suggesting the use of the word "hub".

**Comment by Startome** [comment]
I'll admit your response made me smile. I do wish you were on discord. You'd have seen me asking people if you had an account there because I wanted to mess with you about it. Reading it now my message seems much more serious than intended.

**Comment by jafdy** [comment]
Space Port Hub.

**Comment by endless-sky** [comment]
The reason the button is two words is to make it clearer what keyboard shortcut it uses (P).

The reason it's named the "spaceport" is because in Escape Velocity the place where you picked up special missions was the "Spaceport Bar." But that meant that in E.V. every planet had to have a description of what that bar looked like, and with hundreds of planets there's only so many different descriptions of a bar that you can come up with before they get repetitive and boring.

**Comment by Amazinite** [comment]
Is that really necessary though? P is the second letter in Spaceport, and there are buttons that already use the second letter as their shortcut. e.g. `Missions` while on the map

**Comment by Jeremy-E-Johnson** [comment]
Maybe it would be a good idea to add an underline to the character in the name that corresponds to the keybinding. Such as a p in Spaceport or the d in Depart. These keys can't be remapped anyways so I don't see much danger in adding it.

**Comment by Startome** [comment]
Yeah whether it's renamed or not I agree with that underlining whatever key is mapped

**Comment by jafdy** [comment]
Press ALT. It underlines the hotkey letter.

**Comment by endless-sky** [comment]
I guess it really isn't that hard to figure out the 'p' shortcut, so I just changed it to being one word in the button and in the text that refers to the button (fba0c75).
